### ISSUE TITLE
PDF viewer forgets page scroll position when rotating device, jumps to top of page 1

### ISSUE THREAD

**Issue by cpeterso** [mozilla-mobile/firefox-ios#17917]
## Steps to reproduce
1. Load a long PDF, such as https://curtclifton.net/papers/MoseleyMarks06a.pdf
1. In portrait orientation, scroll down many pages, such as page 10.
1. Rotate the device to landscape orientation. 

### Expected behavior
The PDF viewer should remember the scroll position, such as page 10. Safari remembers the scroll position. 

### Actual behavior
The PDF viewer resets to the top of the PDF, page 1. (Chrome on iOS doesn’t remember the scroll position either.)

### Device & build information
* Device: iPhone 15 Plus, iOS 17.1.2
* Build version: TestFlight build 9000 (36930)
* First seen version: This is not a regression. I can reproduce it in the oldest TestFlight builds I have.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-8030)

**Comment by dicarobinho** [comment]
Hey, @cpeterso,

Do you still have this issue? I can't reproduce it.
https://github.com/mozilla-mobile/firefox-ios/assets/61138287/09ab7ba6-754d-41a0-b80b-893aec80d040

**Comment by cpeterso** [comment]
> Do you still have this issue? I can't reproduce it.

Nope! It works correctly for me in TestFlight Nightly build 9000 (41294), so I'll close this bug. Thanks for testing.

**Comment by data-sync-user** [comment]
➤ Alexandru Farcasanu commented:

Going to close it. Already fixed.

**Comment by cpeterso** [comment]
> what change fixed it?

I don't know which change fixed this PDF. I tested the oldest TestFlight build available on my iPhone (build 38344 from about three months ago) and couldn't reproduce the bug. So either this bug was fixed between the build I reported (36930) and 38344 or this was a bug in iOS's PDF component itself. When I reported this bug four months ago, I had iOS 17.1.2 and now I have iOS 17.4.1.
